### PR TITLE
[NEW] Facebook livechat integration

### PR DESCRIPTION
**Pull request by sampaiodiego**
@RocketChat/core 

This feature will allow livechat agents to respond inquiries from Facebook pages' messenger :tada: 

Just pick the pages you want to respond:
image

And start chatting:
image

### CODE REVIEW

File: packages/rocketchat-livechat/imports/server/rest/facebook.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ * @apiParam {String} page Facebook pages id
+ * @apiParam {String} token Facebook user's token
+ * @apiParam {String} first_name Facebook user's first name
+ * @apiParam {String} last_name Facebook user's last name
+ * @apiParam {String} [text] Facebook message text
+ * @apiParam {String} [attachments] Facebook message attachments
+ */
+RocketChat.API.v1.addRoute('livechat/facebook', {
+	post() {
+		if (!this.bodyParams.text && !this.bodyParams.attachments) {
```
Comment (rodrigok) [inline packages/rocketchat-livechat/imports/server/rest/facebook.js]: Should we check if the request came from facebook via some kind of token?

Comment (rodrigok) [inline packages/rocketchat-livechat/imports/server/rest/facebook.js]: Should check if facebook integration is enabled too.

Comment (sampaiodiego) [reply packages/rocketchat-livechat/imports/server/rest/facebook.js]: yes.. done

Comment (sampaiodiego) [reply packages/rocketchat-livechat/imports/server/rest/facebook.js]: done as well =)

**Comment by gudge25** [comment]
attachments is not working

**Comment by gudge25** [comment]
both  directions

**Comment by geekgonecrazy** [comment]
feel free to open an issue.  This PR is already merged and has been for a while.  Best chance of getting eyes on it would be a new issue

**Pull request merged**